### PR TITLE
ported over www and introduced hugo; closes #1

### LINKED ISSUE #1: [www] Port over www with hugo

### PR DESCRIPTION
**Pull request by thuey**

### CODE REVIEW

File: themes/material-blog/layouts/partials/header.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {{ define "menu-item" }}
 <li>
-    <a class="black-text" href="{{ .URL | relLangURL }}">{{ .Name }}</a>
+    {{ if eq (hasPrefix .URL "mailto:") true }}
```
Comment (thuey) [inline themes/material-blog/layouts/partials/header.html]: I could see just implementing a `header.html` file in the main layout that does this override, but I don't think we should worry about making this theme easy to update when the creator makes changes. Thoughts?

**Comment by thuey** [comment]
@jonnymbgood I went ahead and published this on www.pleaseops.com so you can check it out. I'd like to have a conversation about finalizing colors and a logo at some point.

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (thuey) [inline deploy.sh]: @jonnymbgood I just grabbed this from hugo's tutorial on setting up hugo with GitHub pages, including using a submodule. We could get smarter with removing everything that's not .git or .gitignore, or not using submodules, but I thought this worked well enough, and it's almost midnight :)

**Comment by thuey** [comment]
@jonnymbgood I believe I fixed the formatting and JS issues. This should be ready to review

File: themes/material-blog/layouts/partials/header.html (hunk 1 above)
Comment (jonnymbgood) [reply themes/material-blog/layouts/partials/header.html]: I'm fine with leaving it in here.

File: deploy.sh (hunk 2 above)
Comment (jonnymbgood) [reply deploy.sh]: I'm fine with submodules when they're only used by scripts. If it works it works.

**Pull request merged**